app/src/main/java/org/p2p/wallet/jupiter/JupiterModule.kt

```diff
@@ -181,7 +181,13 @@ object JupiterModule : InjectionModule {
         }
         factoryOf(::SwapTokenRateLoader)
         singleOf(::SwapStateManagerHolder)
-        singleOf(::SwapRateTickerManager)
+        single {
+            SwapRateTickerManager(
+                swapScope = get(),
+                userLocalRepository = get(),
+                swapTokensRepository = get()
```

Comment:
using `single` instead of `singleOf` related to that the SwapRateTickerManager has one more argument which shouldn't be provided by koin